From a16492814a5585f5cc4c4b651f9134c56bc00d98 Mon Sep 17 00:00:00 2001
From: Chen Sai <chensai6@huawei.com>
Date: Mon, 1 Aug 2022 10:24:09 +0800
Subject: [PATCH 4093/4585] codecheck2.0: add cloudbuild2.0 config file for
 codecheck

Offering: HULK
hulk inclusion
category: feature
bugzilla: N/A

--------------------------------

add cloudbuild2.0 config file for codecheck2.0

Signed-off-by: Chen Sai <chensai6@huawei.com>
---
 .cloudbuild/build.yml | 28 ++++++++++++++++++++++++++++
 1 file changed, 28 insertions(+)
 create mode 100644 .cloudbuild/build.yml

diff --git a/.cloudbuild/build.yml b/.cloudbuild/build.yml
new file mode 100644
index 000000000000..f2a94cd23b0b
--- /dev/null
+++ b/.cloudbuild/build.yml
@@ -0,0 +1,28 @@
+---
+version: 2.0
+#....
+env:
+  resource:
+    image: >-
+      kweecr04.his.huawei.com:80/ecr-build-arm-gzkunpeng/csinewtool_1655364810051:hulk-5.10
+    class: 4U4G
+    type: docker
+#....
+steps:
+  PRE_BUILD:
+    - checkout:
+        path: hulk
+  BUILD:
+    - build_execute:
+        command: cd hulk && make olddefconfig && make -j$(nproc)
+        check: 
+          buildcheck:
+            ruleset: default #配置后启用构建规范问题屏蔽能力
+            task_name: ${CLOUD_BUILD_PROJECT_NAME}_buildcheck   #必配项，此参数指定codecheck检查任务名称
+                                                                # ${CLOUD_BUILD_PROJECT_NAME}是环境变量，值为构建工程名称
+                                                                #（可以自定义配置此参数，不会配置就直接复制）
+          compileoption:
+            ruleset: default #配置后启用编译选项规范问题屏蔽能力(C/C++通用部分)
+            task_name: ${CLOUD_BUILD_PROJECT_NAME}_compileoption   #必配项，此参数指定codecheck检查任务名称
+                                                                # ${CLOUD_BUILD_PROJECT_NAME}是环境变量，值为构建工程名称
+                                                                #（可以自定义配置此参数，不会配置就直接复制）
-- 
2.17.1

